tests/files.rs

```diff
@@ -26,6 +26,14 @@ fn generate_tests(glob: &str) -> Vec<Trial> {
                     .unwrap();
             }
 
+            if run.test_type == RunType::CompileBrilLLVM {
```

Comment:
this should probably be the same code path as the if above
you should also generate an args file like the compileBrilift mode for use in nightlies